ansible/roles/chaos/tasks/main.yml

```diff
@@ -0,0 +1,50 @@
+# Chaos
+
+- name: Setup chaos directories
+  file: path="{{ item }}" owner=root group=root mode=0755 state=directory
+  with_items:
+    - "{{ chaos_dir }}"
+    - "{{ chaos_dir }}/start"
+    - "{{ chaos_dir }}/stop"
+  tags:
+    - start
+
+- name: Install chaos scripts
+  sudo: yes
+  template:
+    src: "{{ item }}.j2"
+    dest: "{{ chaos_dir }}/start/{{ item }}"
+    group: root
+    owner: root
+    mode: 0755
+    backup: yes
+  with_items: "{{ enable.split(',') }}"
```

Comment:
this seems like a natural fit for tags, something like `ap deploy_chaos.yml --tags=burnio`. then each chaos script could be its own task. I see that you use tags later on to do stop/start but maybe those belong in separate playbooks. so it would end up `ap start_chaos.yml --tags=burnio` and `ap stop_chaos.yml --tags=burnio`, that also might give slightly more control over what to stop, instead of always stopping everything. I am not sure this idea is strictly better, but passing in an enable variable feels clunky.